src/redis_geo.cc

```diff
@@ -174,7 +174,7 @@ rocksdb::Status Geo::MGet(const Slice &user_key,
 }
 
 std::string Geo::EncodeGeoHash(double longitude, double latitude) {
-  std::string geoalphabet = "0123456789bcdefghjkmnpqrstuvwxyz";
+  const static std::string geoalphabet = "0123456789bcdefghjkmnpqrstuvwxyz";
```

Comment:
CI failed, `static` identifier is unnecessary

Reply:
> src/redis_geo.cc:177:`  Storage-class specifier (static, extern, typedef, etc) should be at the beginning of the declaration.

@git-hulk From cpplint detail, it seems we should change `const static` into `static const`?

Reply:
yeah, or remove the `static` should be OK

Reply:
Hi @git-hulk @byronhe Since we haven't handle with it for a long time, i just remove `static` to resolve this patch